src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/ShellSectionRenderer.cs

```diff
@@ -588,12 +588,22 @@ public override UIViewController[] PopToViewController(UIViewController viewCont
 		public override void PushViewController(UIViewController viewController, bool animated)
 		{
 			_pendingViewControllers = null;
-			base.PushViewController(viewController, animated);
+			if (IsInMoreTab && ParentViewController is UITabBarController tabBarController)
+			{
+				tabBarController.MoreNavigationController.PushViewController(viewController, animated);
```

Comment:
Is pushing from the more tab a typical iOS thing? I have not been able to find an Apple app that does the more button, so I am not sure. In my mind the More tab is like a popup, but I suppose it does not look like it. However, since I have no idea what it is supposed to look like.

But, I am also not sure it really matters as the More tab is frowned upon so you should not be using it.